documentation/asciidoc/computers/linux_kernel/configuring.adoc

```diff
@@ -1,49 +1,54 @@
+Once you've got everything set up, you can compile and run the `menuconfig` utility as follows:
 
-[,bash]
+[source,console]
 ----
- make menuconfig
+$ make menuconfig
 ----
 
-If you're cross-compiling a 32-bit kernel:
+To cross-compile a 64-bit kernel:
 
-[,bash]
+[source,console]
 ----
-make ARCH=arm CROSS_COMPILE=arm-linux-gnueabihf- menuconfig
+$ make ARCH=arm64 CROSS_COMPILE=aarch64-linux-gnu- menuconfig
 ----
 
-Or, if you are cross-compiling a 64-bit kernel:
+To cross-compile a 32-bit kernel:
 
-[,bash]
+[source,console]
 ----
-make ARCH=arm64 CROSS_COMPILE=aarch64-linux-gnu- menuconfig
+$ make ARCH=arm CROSS_COMPILE=arm-linux-gnueabihf- menuconfig
 ----
 
-The `menuconfig` utility has simple keyboard navigation. After a brief compilation, you'll be presented with a list of submenus containing all the options you can configure. There are many options, so take your time to read through them and get acquainted.
+To navigate the `menuconfig` utility, use your keyboard:
 
-Use the arrow keys to navigate, the Enter key to enter a submenu (indicated by `+--->+`), Escape twice to go up a level or exit, and the space bar to cycle the state of an option. Some options have multiple choices, in which case they'll appear as a submenu and the Enter key will select an option. You can press `h` on most entries to get help about that specific option or menu.
+* to navigate directionally, use the *arrow keys*
+* to enter a submenu (indicated by `--->`), press the *Enter* key
+* to go up a level or exit, press *Escape* twice
+* to cycle the state of a binary option, press the *space bar*
+* to select the state of a multiple choice option, press *Enter* to open a submenu, the *arrow keys* to navigate the submenu, and press *Enter* again to select a state
+* to get help about with an option or menu, press the *H* key
 
-Resist the temptation to enable or disable a lot of things on your first attempt; it's relatively easy to break your configuration, so start small and get comfortable with the configuration and build process.
+After a brief compilation, `menuconfig` presents a list of submenus containing all the options you can configure. There are many options, so take your time to read through them. Resist the temptation to enable or disable a lot of things on your first attempt; it's relatively easy to break your configuration, so start small and get comfortable with the configuration and build process.
 
-=== Saving your changes
+=== Save your changes
 
-Once you're done making the changes you want, press Escape until you're prompted to save your new configuration. By default, this will save to the `.config` file. You can save and load configurations by copying this file.
+Once you're done making changes, press *Escape* until you're prompted to save your new configuration. By default, this saves to the `.config` file. You can save and load configurations by copying this file.
```

Comment:
I wonder if (for completeness) the end of this section should then link back to the kernel-building instructions?